### PR TITLE
tests using puppet resource to create and describe ec2_securitygroup

### PR DESCRIPTION
**Pull request by zreichert**
This PR requires code that is in https://github.com/puppetlabs/puppetlabs-aws/pull/46

### CODE REVIEW

File: spec/acceptance/securitygroup_spec.rb [hunk 1]
```diff
@@ -276,4 +276,106 @@ def expect_rule_matches(ingress_rule, ip_permission)
...
+        regex = /(description)(\s*)(=>)(\s*)('#{@config[:description]}')/
+        expect(@response.stdout).to match(regex)
+      end
+
+      it 'region is correct' do
+        regex = /(region)(\s*)(=>)(\s*)('#{@config[:region]}')/
+        expect(@response.stdout).to match(regex)
+      end
+
+      it 'tags are correct'
```
Comment (garethr) [inline spec/acceptance/securitygroup_spec.rb]: missing a `do`

**Comment by garethr** [comment]
@zreichert I've merged the helpers but spotted a quick syntax error. If you could rebase with master and fix I'll get this merged too.

**Comment by zreichert** [comment]
@garethr Typo fixed and rebased.  That test is still marked as pending.  Is that correct?

**Comment by garethr** [comment]
@zreichert yes, I'll add tags to everything based on the work in the VPC branch

**Pull request merged**